packages/config/src/layer2s/degate.ts

```diff
@@ -0,0 +1,265 @@
+import { assert, ProjectId, UnixTime } from '@l2beat/shared'
+import { utils } from 'ethers'
+
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import { formatSeconds } from '../utils/formatSeconds'
+import {
+  DATA_AVAILABILITY,
+  EXITS,
+  FORCE_TRANSACTIONS,
+  makeBridgeCompatible,
+  NEW_CRYPTOGRAPHY,
+  OPERATOR,
+  RISK_VIEW,
+  STATE_CORRECTNESS,
+} from './common'
+import { Layer2 } from './types'
+
+const discovery = new ProjectDiscovery('degate')
+
+const forcedWithdrawalDelay = formatSeconds(
+  discovery.getContractValue<number[]>('ExchangeV3', 'getConstants')[2],
+)
+const maxAgeDepositUntilWithdrawable = formatSeconds(
+  discovery.getContractValue<number>(
+    'ExchangeV3',
+    'getMaxAgeDepositUntilWithdrawable',
+  ),
+)
+
+const forcedWithdrawalFee = discovery.getContractValue<number>(
+  'LoopringV3',
+  'forcedWithdrawalFee',
+)
+
+const forcedWithdrawalFeeString = `${utils.formatEther(
+  forcedWithdrawalFee,
+)} ETH`
+
+export const degate: Layer2 = {
+  type: 'layer2',
+  id: ProjectId('degate'),
+  display: {
+    name: 'DeGate',
+    slug: 'degate',
+    description:
+      'DeGate is an app-specific ZK rollup that enables a Trustless, fast and low-fee decentralized order book exchange, helping users to trade easy and sleep easy.',
```

Comment:
```suggestion
      'DeGate is an app-specific ZK rollup that enables a trustless, fast and low-fee decentralized order book exchange, helping users to trade easy and sleep easy.',
```

Reply:
I would also add that it is a fork of Loopring, to give context the same way we do for example for Boba and Metis